examples/device/cdc_msc/src/main.c

```diff
@@ -105,34 +105,31 @@ void tud_resume_cb(void)
 //--------------------------------------------------------------------+
 void cdc_task(void)
 {
-  if ( tud_cdc_connected() )
+  // is data available to read from rx fifo
```

Comment:
I still prefer to have `tud_cdc_connected()` in example code. Since it is still the norm for most terminal rather than not. Only a few terminal does not do this. This API is very handy in many scenarios e.g allow cdc-device to stay in lower power mode etc ... Since I don't write enough documentation, the example serves this purpose so yeah, please keep it.